### ISSUE TITLE
[Bug]: "Disable Autoplay" plugin doesn't work when "Applies only on startup" is enabled

### ISSUE THREAD

**Issue by soarpasser** [th-ch/youtube-music#1821]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.2

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Win10 22H2

### What CPU architecture are you using?

x64

### Reproduction steps

Enable "Disable Autoplay" plugin and "Apply only on startup"
Play any song
Restart the application

### Expected Behavior

The last played song should be paused when the application is restarted.

### Actual Behavior

When starting the application, the last song will still play normally instead of pausing. 
However when the "Applies only on startup" function is set to OFF, then the plugin behaves as expected.

### Enabled plugins

For easier troubleshooting I have attached the config.json file directly.
[config.json](https://github.com/th-ch/youtube-music/files/14473110/config.json)

**Comment by rfgamaral** [comment]
Yeah, it's a bit annoying restarting the computer in the middle of the night, and have the app autoplaying something and waking up everyone 😅 For me, it only plays for like a second or so, but it's still annoying.

> However when the "Applies only on startup" function is set to OFF, then the plugin behaves as expected.

Not for me. It always autoplays regardless of the `Applies only on startup` option value.

---

@JellyBrick Do you think that it's possible to do something about this?

**Comment by SomeAspy** [comment]
cannot reproduce.
seems to function as expected
@rfgamaral open a new issue, don't hijack issues
